### PR TITLE
Adding support for webp images

### PR DESCRIPTION
**Pull request by nbsmith**
Adds support for being able to download webp images and convert as normal to preferred format.

### CODE REVIEW

File: pkg/core/core.go [hunk 1]
```diff
@@ -294,7 +294,12 @@ func (comic *Comic) DownloadImages(options *config.Options) (string, error) {
 			}
 			defer imgFile.Close()
 
-			err = util.SaveImage(imgFile, rsp.Body, format)
+			if strings.HasSuffix(link, ".webp") {
```
Comment (Girbons) [inline pkg/core/core.go:297]: here we can do something like:

```go      
isWebp := strings.HasSuffix(link, ".webp")
err = util.SaveImage(imgFile, rsp.Body, format, isWebp)
```

File: pkg/util/image.go [hunk 2]
```diff
@@ -55,3 +59,26 @@ func SaveImage(w io.Writer, content io.Reader, format string) error {
 		return errors.New("format not found")
 	}
 }
+
+func SaveImageWEBP(w io.Writer, content io.Reader, format string) error {
```
Comment (Girbons) [inline pkg/util/image.go:63]: here we can reuse `SaveImage` 

```go
func SaveImage(w io.Writer, content io.Reader, format string, isWebp bool) error {
  var (
    img image.Image
    err error
  )

	if isWebp {
		img, err = webp.Decode(content)
	} else {
		img, _, err = image.Decode(content)
	}
       ...
```

**Review by Girbons — CHANGES_REQUESTED** [review]
thanks for the contribution @nbsmith! I just left a couple of notes before merging

**Comment by nbsmith** [comment]
Looks good. Made the changes and it looks like they work just fine.

**Comment by nbsmith** [comment]
Looked at the dependabot pull requests. Did those + the sys and text modules. It compiles and properly converts webp.

**Pull request merged**